pkg/plugins/hybrid/v1alpha/scaffolds/init.go

```diff
@@ -32,7 +32,7 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin/util"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
 
-	kustomizev2alpha "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/kustomize/v2-alpha"
+	kustomizev2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/kustomize/v2"
```

Comment:
Could you run `make generate` too to re-scaffold out the testdata. I'm sure there maybe some breaking changes coming in.